### PR TITLE
# Added comments and readability updates to rain and snow notebooks

### PR DESCRIPTION
**Pull request by rectheworld**
Changes:

1. Added comments for readability and understanding
2. Added more QA cells for counting rows and circles
3. Added printouts for conditional breakouts for use in the powerpoint deliverable

### CODE REVIEW

File: notebooks/2.1.1-he-rain-analysis.ipynb [hunk 1]
```diff
@@ -72,22 +72,31 @@
    "metadata": {},
```
Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>should be "the below line is <strong>the percent of circles </strong>who had at least one station reporting but those stations could not agree"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='100'/>

Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>also should be a percentage of circles, not count of stations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>

Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>same as above: percentage of circles whose stations could not agree</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='106'/>

Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>The second and third shouldnt be consensus, but just what the station closest in location/elevation reported for circles that reported rain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='110'/>

Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>same as above: second and third are not consensus values (i didnt calculate consesus for altitude/distance) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='112'/>

Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>this just station data all by itself (station data vs station data)</p><p>testing to see if there is that much difference between the "truth" from the different station metrics (ie, consensus, consensus amongst the same ecosystem)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='157'/>

Comment (hamzaelsaawy) [inline notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>same as above, not volunteer data here</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='168'/>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

Comment (rectheworld) [reply notebooks/2.1.1-he-rain-analysis.ipynb:72]: <p>Fixed in Rain, snow, and snowfall notebooks</p>

**Comment by rectheworld** [comment]
@hamzaelsaawy I made the comment edits you requested on all the notebooks that were copied from the rain notebook.

**Pull request merged**